workspaces/defectdojo/plugins/defectdojo-backend/src/setupTests.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Setup file for Jest tests
+import '@testing-library/jest-dom';
+
+// Mock global fetch for tests
+global.fetch = jest.fn();
```

Comment:
why? 🤔 
This is a bit dangerous as if we forget to mock some requests, we might get inconsistent tests.